proto/autokitteh/events/v1/svc.proto

```diff
@@ -29,6 +29,10 @@ message SaveResponse {
 
 message GetRequest {
   string event_id = 1 [(buf.validate.field).string.min_len = 1];
+
+  // If set, all values returned will be string values
+  // that contained the native values in JSON format.
```

Comment:
1. s/contained/contain/
2. Explain also what happens if the bool is `false`

Reply:
done